examples/cra-ts-kitchen-sink/package.json

```diff
@@ -47,7 +47,7 @@
     "enzyme-adapter-react-16": "^1.9.1",
     "enzyme-to-json": "^3.4.1",
     "fork-ts-checker-webpack-plugin": "^4.0.3",
-    "react-docgen-typescript-plugin": "^0.4.0",
```

Comment:
Do we need this dep at all? i made RDTP a hard dep in the CRA preset because it was breaking with SB 5.3 instances.

Reply:
And it's already a dep in `@storybook/react`